resources/lang/en/beatmaps.php

```diff
@@ -168,6 +168,8 @@
         'search' => [
             'prompt' => 'type in keywords...',
             'options' => 'More Search Options',
+            'supporter-filter' => 'this filter requires supporter',
```

Comment:
change "this filter" to either "this filter combination" or ":filter_names filter" (with `filter_names` being the list of filters).